v3/src/components/data-summary.tsx

```diff
@@ -28,9 +30,34 @@ export const DataSummary = observer(({ broker, v2Document }: IProps) => {
     setSelectedAttribute(data?.attrFromID(attributeId))
   }
 
+  const handleDataSetSelection = (evt: React.ChangeEvent<HTMLSelectElement>) => {
+    broker?.setSelectedDataSetId(evt.target.value)
+  }
+
+  const DataSelectPopup = () => {
+    const dataSetSummaries = broker?.summaries
+    const renderOption = (name: string, id: string) => {
+      return <option key={name} value={id}>{name}</option>
+    }
+
+    if (dataSetSummaries) {
+      return (
+        <Select onChange={handleDataSetSelection} value={data?.id}>
+          { dataSetSummaries?.map(summary => {
+              return renderOption(summary.name || `DataSet ${summary.id}`, summary.id)
+            })
+          }
+        </Select>
+      )
+    } else {
+      return null
+    }
```

Comment:
```suggestion
    return null
```
You don't need the `else` here.